cdap-ui/app/cdap/components/HttpExecutor/RequestHistoryTab/index.tsx

```diff
@@ -284,10 +310,26 @@ const RequestHistoryTabView: React.FC<IRequestHistoryTabProps> = ({
     }
   };
 
+  const getTotalLogCount = () => {
+    let count = 0;
+    requestLog
+      .keySeq()
+      .toArray()
+      .forEach((dateID: string) => {
+        count += requestLog.get(dateID).size;
+      });
```

Comment:
Could you do `requestLog.values().forEach((v) => { count += v.size; });`? This would save a conversion and a lookup.

Reply:
Because I've updated the code from previous related PRs, we no longer need to look up values